files/docker-container-logrotate

```diff
@@ -1,9 +1,11 @@
 /var/lib/docker/containers/*/*.log {
   rotate 0
-  daily
+  hourly
```

Comment:
Alright! As per offline discussion, parameterized the logrotation timer to run on a 15 mins cadence. As for log config, logrotate service will rotate it if it exceeds 10MB, or else it will anyways rotate it after each hour.